### ISSUE TITLE
common.sh's function get_test_vm_list() breaks on Satellite 6.2

### ISSUE THREAD

**Issue by pcfe** [RedHatSatellite/soe-ci#50]
common.sh uses

```
"hammer content-host list --organization \"${ORG}\" \
--host-collection \"$TESTVM_HOSTCOLLECTION\" \
```

While this works fine on Satellite 6.1, Satellite 6.2's hammer has changed in that regard.

```
# hammer content-host list --organization "Sat Test" --host-collection "Test Servers RHEL7"
Error: Unrecognised option '--host-collection'

See: 'hammer content-host list --help'
```

**Comment by pcfe** [comment]
on Sat 6.2
```
"hammer host-collection hosts --organization \"${ORG}\" \
--name \"$TESTVM_HOSTCOLLECTION\" \
```
 works
```
[root@satellite-6.2 ~]# hammer host-collection hosts --name "Test Servers RHEL7" --organization-id 1
---|---------------------------------
ID | NAME                            
---|---------------------------------
2  | sattestclient01.sattest.pcfe.net
4  | sattestclient02.sattest.pcfe.net
---|---------------------------------
```
but I need to verify first if this also works on 6.1 before making the code change.

**Comment by pcfe** [comment]
note to self: check the following in `pushtests.sh` on Sat 6.1.

on Sat 6.2 'Checking if test server ... has rebuilt... ' does not pick up that the machine is done installing and ready for testing.

```
+ echo -n 'Checking if test server sattestclient02.sattest.pcfe.net has rebuilt... '
Checking if test server sattestclient02.sattest.pcfe.net has rebuilt... ++ ssh -q -l jenkins -i /var/lib/jenkins/.ssh/id_rsa satellite.internal.pcfe.net 'hammer host info --name sattestclient02.sattest.pcfe.net |             grep -e "Managed.*true" -e "Enabled.*true" -e "Build.*false" 		| wc -l'

+ status=0
+ [[ 0 == 3 ]]
+ tell 'Not yet.'
+ fold -s
++ date
+ echo 'Not yet. [Sat Jul 30 21:25:03 UTC 2016]'
Not yet. [Sat Jul 30 21:25:03 UTC 2016]
+ [[ 1850 -gt 6000 ]]
+ [[ 2 -gt 0 ]]
+ sleep 10
```

```
bash-4.2$ ssh -q -l jenkins -i /var/lib/jenkins/.ssh/id_rsa satellite.internal.pcfe.net 'hammer host info --name sattestclient02.sattest.pcfe.net |             grep -e "Managed.*true" -e "Enabled.*true" -e "Build.*false" | wc -l'
0
bash-4.2$ ssh -q -l jenkins -i /var/lib/jenkins/.ssh/id_rsa satellite.internal.pcfe.net 'hammer host info --name sattestclient02.sattest.pcfe.net'
Id:                 4
Name:               sattestclient02.sattest.pcfe.net
Organisation:       Sat Test
Location:           Bergmannstraße
Host Group:         hg-test-VMs-library
Environment:        KT_Sat_Test_Library_cv_Jenkins_SOE_el7_6
Puppet CA Id:       1
Puppet Master Id:   1
Cert name:          sattestclient02.sattest.pcfe.net
Managed:            yes
Installed at:       2016/07/30 21:11:19
Last report:        2016/07/30 21:12:45
Network:            
    IP:     192.168.200.102
    MAC:    52:54:00:50:22:f5
    Subnet: Satellite Test Network
    Domain: sattest.pcfe.net
Network interfaces: 
 1) Id:          5
    Identifier:  ens3
    Type:        interface (primary, provision)
    MAC address: 52:54:00:50:22:f5
    IP address:  192.168.200.102
    FQDN:
Operating system:   
    Architecture:           x86_64
    Operating System:       RedHat 7.2
    Build:                  no
    Partition Table:        Kickstart default
    Custom partition table:
Parameters:         
    kt_activation_keys => ak-Jenkins-SOE-el7
Additional info:    
    Owner Id:   3
    Owner Type: User
    Enabled:    yes
    Model:      KVM
    Comment:

bash-4.2$ 
```

check if the hammer command in both sat versions can maybe spit out 1 | 0 instead of text for _Managed_ and _Enabled_.

**Comment by pcfe** [comment]
note to self:
`common.sh` does
```
                if [[ "${CONDITIONAL_VM_BUILD}" != 'true' ]] || \
                        [[ ! -s "${MODIFIED_CONTENT_FILE}" ]] || \
                        ssh -q -l ${PUSH_USER} -i ${RSA_ID} ${SATELLITE} \
                        "hammer --output yaml host info --name \"${I}\"" \
                                | grep "^Comment:" \
                                | grep -Fqf "${MODIFIED_CONTENT_FILE}"
                then
                        TEST_VM_LIST[$J]=$I
                        ((J+=1))
                fi
```

```
-F, --fixed-strings, --fixed-regexp
       Interpret  PATTERN  as  a list of fixed strings, separated by newlines, any of which is to be matched.
       (-F is specified by POSIX, --fixed-regexp is an obsoleted alias, please do not use it in new scripts.)
```

Be sure not to break that, if changing format of MODIFIED_CONTENT_FILE

**Comment by pcfe** [comment]
cleaner attempt in branch [sat6.2-changes](https://github.com/RedHatSatellite/soe-ci/tree/sat6.2-changes)
still not perfect, but better than what I had in the branch ISSUE-50 (which has been removed)

**Comment by pcfe** [comment]
fixed by 8d49d96
